product_custom_info/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================
+Product Custom Info
+===================
+
+This module extends products to allow having custom properties on them.
+
+Installation
+============
+
+* Install `OCA/server-tools <https://github.com/OCA/server-tools>`_.
+
+Usage
```

Comment:
It lacks section Configuration to indicate where you create the Custom Info Templates.